Source/WTF/wtf/glib/ChassisType.cpp

```diff
@@ -43,7 +43,7 @@ static std::optional<ChassisType> readMachineInfoChassisType()
     }
 
     GUniquePtr<char*> split(g_strsplit(buffer.get(), "\n", -1));
-    WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
+    WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // GLib port.
```

Comment:
Not sure this speaks for itself; I don't fully understand why emphasizing which port this code is specific to belongs in a comment on the macro invocation line. I don't have an objection to it if it's helpful but I don't know the rationale.

Reply:
I'm following the pattern previously established by @geoffreygaren. The idea is to help identify remaining `WTF_ALLOW_UNSAFE_BUFFER_USAGE` usage that impacts Apple ports (vs not).